app/models/garden_health.rb

```diff
@@ -1,5 +1,6 @@
 class GardenHealth < ApplicationRecord
   validates :reading_type, :reading, presence: true
+  validates :reading, numericality: true
 
   belongs_to :sensor, optional: true
 
```

Comment:
Is there a reason we didn't initially validate the presence of time_of_record?

Reply:
We decided not to validate it because we wanted the user to have the ability to add the time of reading later on in the case of them manually inputing data. Now that I am reflecting on this, I am not sure if that is the best route but that was the thought at the time.

Reply:
time_of_record would only be used when a user creates data manually. Otherwise when data is uploaded from the sensor we didn't want it to be validated because timestamps was already taking care of by the automated data input.

Reply:
I see. I will leave it as is then!